server/routes/api.js

```diff
@@ -27,6 +27,23 @@ router.get('/asset-compute-endpoint', async function(req, res) {
     });
 });
 
+
+router.get('/setUpDevToolGetJournalUrl', async function(req, res) {
```

Comment:
Probably good to stay consistent with the naming pattern here, e.g. `/asset-compute-endpoint` or `/asset-compute-action-url`.

(Also it makes my REST heart cringe a little when I see RPC style method names in URLs 😉 )